src/bud/api/map.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * ## bud.map
+ *
+ * Enable or disable source-maps
+ *
+ * ### Example
+ *
+ * ```js
+ * bud.map(true)
+ * ```
+ */
+const map: Map = function (enabled: boolean): Bud {
+  this.state.features.map = enabled
+
+  return this
+}
+
+export {map}
+
+import type {Bud} from '..'
+export type Map = (enabled: boolean) => Bud
```

Comment:
I might just be a bit unfamiliar with TypeScript, but wouldn't this conflict with JavaScript's `Map`? From the perspective of someone who is learning the codebase it could seem a bit confusing.

Reply:
this is now `SourceMap`